docs/serving/services/storage.md

```diff
@@ -1,12 +1,35 @@
 # Volume Support for Knative services
 
-By default Serving supports the mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`. [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](../configuration/feature-flags.md) to be enabled.
+You can provide data storage for Knative Services by configuring different types of volumes.
```

Comment:
`four` volume types or do you see projected as another definition of the three?

Reply:
Actually I see them as four given that at the K8s api level we have:
```
Volume -> (name, VolumeSource)

VolumeSource -> {EmptyDir, Secret, ConfigMap, Projected, DownwardAPI,...}

Projected ->  {DownwardAPIProjection, ConfigMapProjection, SecretProjection, ServiceAccountProjection }

```
Note that volumeSource `DownwardAPI` is [not supported directly it seems](https://github.com/knative/serving/blob/main/pkg/apis/serving/k8s_validation.go#L187-L202).
The last [PR](https://github.com/knative/serving/pull/13896) on this area was btw the addition of the projected downwardapi.

Reply:
Ah, but then the listing is a bit misleading. 

> different types of volumes

and in the next sentence:

> [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`

Can we change one of those "types" then?

Reply:
These are different type of volumes according to the api.